src/ol/style/expressions.js

```diff
@@ -21,15 +21,18 @@ import {asArray, fromString, isStringColor} from '../color.js';
  *     of bands, depending on the underlying data source and
  *     {@link import("../source/GeoTIFF.js").Options configuration}. `xOffset` and `yOffset` are optional
  *     and allow specifying pixel offsets for x and y. This is used for sampling data from neighboring pixels.
- *   * `['get', 'attributeName']` fetches a feature attribute (it will be prefixed by `a_` in the shader)
+ *   * `['get', 'attributeName', typeHint]` fetches a feature attribute (it will be prefixed by `a_` in the shader)
  *     Note: those will be taken from the attributes provided to the renderer
+ *     A type hint can optionally be specified, in case the resulting expression contains a type uncertainty which
+ *     will make it invalid. Type hints can be one of: 'string', 'color', 'number', 'boolean', 'number_array'
```

Comment:
I think `number_array` might be the first introduction of snake_case to the API.

What do you think about these alternatives:
 * `numbers`
 * `numberArray`

Reply:
I would go with `numberArray`; on the other hand I considered `number[]`, it's shorter and quite explicit for more people (because of typescript).

Reply:
I think `number[]` sounds good.